src/helper.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Given a domain and endpoint string, return a correctly formatted url string
+ * e.g.
+ * test.practera.com - login => https://test.practera.com/login
+ *
+ */
+export const urlFormatter = (domain: string, endpoint?: string) => {
```

Comment:
@sasangachathumal i understand we use `helper.ts` file in our loosely structured project, our angular app has obvious file structure to follow. This utility-related function should placed inside UtilsService file.

Reply:
sure

Reply:
@trtshen I have moved 'urlFormatter' to the utilService.